cellprofiler/modules/dilatepixels.py

```diff
@@ -1,10 +1,10 @@
 # coding=utf-8
 
 """
-Dilation
+DilatePixels
```

Comment:
reST will complain if the ===s isn't the same length as the line above. Please adjust the length of this line appropriately.